import/import.go

```diff
@@ -185,27 +185,33 @@ size: estimated file size
 addTags: Add a list of tags to all images
 fetchTags: fetch tags from gelbooru
 */
-func ImportFile(f io.ReadSeeker, size int, addTags string, fetchTags bool,
+func ImportFile(f io.ReadSeeker, size int, name string, addTags string, fetchTags bool,
 ) (r common.Image, err error) {
 	ch := make(chan response)
 	importFile <- request{f, size, addTags, ch}
 	res := <-ch
 	r = res.Image
 	err = res.err
+
 	if err != nil {
 		return
 	}
 
 	if fetchTags && fetch.CanFetchTags(r.Type) {
 		var fetched []common.Tag
 		fetched, err = fetch.FetchTags(r.MD5)
+
 		if err != nil {
 			return
 		}
+
+		fetched = append(fetched, tags.Normalize(name, common.User))
```

Comment:
No longer needed. Need to set `r.Name` instead. Also a good idea to trim the name, if too long. Something like 100 chars should do.